tests/utls/test-transport.ts

```diff
@@ -0,0 +1,15 @@
+import * as Transport from "winston-transport";
+import {LogEntry} from "winston";
+
+export class TestTransport extends Transport {
```

Comment:
Could you elaborate what this is needed for? I'm not sure I'm following :D

Reply:
I am using this to capture logs during testing. By attaching a custom transport (similar to [this](https://github.com/celonis/content-cli/blob/master/src/util/logger.ts#L24)) I can then read the logs as you would e.g. in Java by adding a TestAppender to logback.